homeassistant/components/plant/__init__.py

```diff
@@ -363,11 +356,11 @@ class DailyHistory:
     At the moment only the maximum value per day is kept.
     """
 
-    def __init__(self, max_length):
+    def __init__(self, max_length) -> None:
```

Comment:
Unrelated to the move of the constants (and all arguments would need to be typed).

Reply:
The thing is that the git hooks don't let me commit until this is fixed.

Reply:
Then there is a problem with your git hooks.
`pylint` should display an error inside VSCode, but it shouldn't block commit.

If you cannot resolve that, you should be able to bypass the git hooks (`git commit -n)` temporarily until you fix the root cause.